components/Layout/Layout.module.scss

```diff
@@ -0,0 +1,16 @@
+.layout {
+  nav {
```

Comment:
We'll want to target these more safely. Since it will hit any button or nav inside the layout container.

Reply:
Oh definitely, I just did this because I expect us to throw it all away but happy to change it!

Reply:
Reckon it's worth it, just in case it's built upon and copying and pasting happens. Only need `.nav` and `.button`.

Reply:
Done